core/vm/contracts.go

```diff
@@ -121,7 +122,14 @@ var PrecompiledContractsBLS = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{18}): &bls12381MapG2{},
 }
 
+// PrecompiledContractsNapoli contains the precompiled Ethereum
+// contract specified in EIP-7212. This is exported for testing purposes.
+var PrecompiledContractsNapoli = map[common.Address]PrecompiledContract{
+	common.BytesToAddress([]byte{19}): &p256Verify{},
+}
```

Comment:
Hello! Thanks for the very first implementation. I just wanted to drop a note:
- Currently address in the spec has not finalized yet. 
- It will be probably discussed and finalized at the next RollCall (# 1) for the RIP purpose.
- We have an unmerged EIP [PR](https://github.com/ethereum/EIPs/pull/7970) to change the implementation address to `0x0b` which is the actual next address.

Reply:
The address has been updated to `0x100` by changing the [PIP](https://github.com/maticnetwork/Polygon-Improvement-Proposals/pull/229) and following the original RIP.

Reply:
Yup, I’ll update it. Is there anything else that needs a modification?

Reply:
I am not sure if there is any Polygon specific change is needed, but rest of it looks good to me. Other than that, even if the PR on geth has been reviewed from some of the core devs, it would be good to have more code reviews, security checks, and more test cases. My PR was to show a reference, might not be ready for the production.

Reply:
Thanks @ulerdogan for the feedback, I've added a few positive and negative testcases, implementation wise it looks good to me as it follows geth standards which we try to follow as well.